test/Maestro.ScenarioTests/TestParameters.cs

```diff
@@ -177,12 +177,17 @@ private TestParameters(
 
     public string AzureDevOpsProject { get; } = "internal";
 
-    public string AzDoToken { get; }
+    public AzureDevOpsTokenProvider AzdoTokenProvider { get; }
```

Comment:
can this be private, since we have the method below?

Reply:
Yeah, it can be, will change